### PR TITLE
#3845 Attempt to add insert, page up and page down keys

### LINKED ISSUE #3845: cy.type('{insert}') is not supported
### Current behavior:

```
CypressError: Special character sequence: '{insert}' is not recognized. Available sequences are: {selectall}, {del}, {backspace}, {esc}, {{}, {enter}, {leftarrow}, {rightarrow}, {uparrow}, {downarrow}, {home}, {end}, {alt}, {option}, {ctrl}, {control}, {meta}, {command}, {cmd}, {shift}
```

### Desired behavior:

`{insert}` should be a supported key. Not sure why it isn't. For that matter, `{pageup}` and `{pagedown}` should also be supported.

### Versions

3.2.0

### PR DESCRIPTION
**Pull request by clarmso**
Here is my attempt to add the support for the insert, page up and page down keys. I've used the PR for #3071 as a guide.

Please feel free to comment on this PR.  I think that these keys behave in a similar way as the escape key: They don't change the input or the cursor's location.

Close #3845.

### CODE REVIEW

**Comment by jennifer-shehane** [comment]
We'll want to open a PR for the documentation to add these to the special chars table upon release. https://github.com/cypress-io/cypress-documentation/blob/develop/source/api/commands/type.md

File: packages/driver/src/cypress/keyboard.coffee [hunk 1]
```diff
@@ -103,6 +106,19 @@ $Keyboard = {
 
         return
 
+    ## charCode = 45
+    ## no keyPress
+    ## no textInput
+    ## yes input (if value is actually changed)
```
Comment (Bkucera) [inline packages/driver/src/cypress/keyboard.coffee]: should this be 'no input'?

**Comment by clarmso** [comment]
I have created a PR on cypress-documentation for these 3 new keys.
https://github.com/cypress-io/cypress-documentation/pull/1567

**Comment by brian-mann** [comment]
@jennifer-shehane is saying that the browser respects the default behavior of pageup + pagedown which is hard to believe... but regardless there should be a test ensuring that this is correctly happening.

**Pull request merged**